lms/js_config_types.py

```diff
@@ -113,6 +113,21 @@ class APIAssignments(TypedDict):
     pagination: NotRequired[Pagination]
 
 
+class RosterEntry(TypedDict):
+    student: APIStudent
+
+    active: bool
+    "Whether or not this student is active in the course/assignment or roster."
```

Comment:
It would be less disruptive if we can make this have the same properties as APIStudent + active, rather than encapsulating a student.

```suggestion
class RosterEntry(APIStudent):
    active: bool
    "Whether or not this student is active in the course/assignment or roster."
```

I also think the resulting JSON is easier to walk and flatten this way, but happy to discuss if you think there's a good reason to use this approach.

Reply:
No, you are right, I'll test that out!

Reply:
Changed to that style